### PR TITLE
Added Pointer on logo text

### PR DESCRIPTION
**Pull request by Vishv07**

### CODE REVIEW

File: simplq/src/styles/adminPage.module.scss [hunk 1]
```diff
@@ -129,5 +129,6 @@
   }
   p {
     font-family: $header-font;
+    cursor: pointer;
```
Comment (daltonfury42) [inline simplq/src/styles/adminPage.module.scss:132]: we should add to the img above too.

Comment (Vishv07) [reply simplq/src/styles/adminPage.module.scss:132]: Yes we can

Comment (Vishv07) [reply simplq/src/styles/adminPage.module.scss:133]: Okk I added , Have a look

**Review by daltonfury42 — APPROVED** [review]

**Comment by daltonfury42** [comment]
Thanks for the PR. And enjoy your tshirt :)

**Pull request merged**